src/pages/workspaces/workspace/Analyses.js

```diff
@@ -232,7 +239,7 @@ const Analyses = _.flow(
   const [copyingAnalysisName, setCopyingAnalysisName] = useState(undefined)
   const [deletingAnalysisName, setDeletingAnalysisName] = useState(undefined)
   const [exportingAnalysisName, setExportingAnalysisName] = useState(undefined)
-  const [sortOrder, setSortOrder] = useState(() => StateHistory.get().sortOrder || defaultSort.value)
+  const [sortOrder, setSortOrder] = useState(() => getLocalPref(KEY_ANALYSES_SORT_ORDER) || defaultSort.value)
```

Comment:
Not strictly related to this PR, but it looks like `defaultSort.label` is unused and `defaultSort` could be simplified.